packages/button/src/css/index.js

```diff
@@ -49,38 +46,23 @@ export default {
     fontSize: type.fontSizeSmall,
     fontWeight: type.fontWeightMedium,
     textAlign: 'center',
-    color: colors.white,
-    background: colors.orange,
     whiteSpace: 'nowrap',
     textDecoration: 'none',
     transition: `all ${motion.speedNormal}`,
     verticalAlign: 'middle',
 
     '&:not([disabled]):hover': {
-      background: colors.orangeLight,
       cursor: 'pointer'
     },
     '&:focus': {
-      outline: 'none',
-
-      '&:before': {
-        content: ' ',
-        position: 'absolute',
-        top: '-4px',
-        left: '-4px',
-        right: '-4px',
-        bottom: '-4px',
-        border: `3px solid ${colors.blue}`,
-        borderRadius: '4px'
-      }
+      outline: 'none'
     }
```

Comment:
we lost our focus state

Reply:
@sweeting what's that blue suppose to be?